examples/gatsbygram/cypress/integration/home-page-spec.js

```diff
@@ -152,33 +154,33 @@ describe(`The Home Page`, () => {
         .click()
 
       // wait for page to transition
-      cy.waitForRouteChange()
+      cy.waitForAPI(`onRouteUpdate`)
         .url()
         .should(`contain`, post1.id)
 
       // press right arrow to go to 2nd post
       cy.get(`body`).type(`{rightarrow}`)
 
       // wait for page to transition
-      cy.waitForRouteChange()
+      cy.waitForAPI(`onRouteUpdate`)
         .url()
         .should(`contain`, post2.id)
 
       // reload the page and go back
       cy.reload()
-        .waitForRouteChange()
+        .waitForAPI(`onRouteUpdate`)
         .go(`back`)
 
       // test if the first post exists
-      cy.waitForRouteChange()
+      cy.waitForAPI(`onRouteUpdate`)
```

Comment:
(ha: basically what @pieh said)

Reply:
cc @pieh I think it would make more sense just to bump the version, since the package currently isn't published to NPM (apart from [my initial test version](https://www.npmjs.com/package/cypress-gatsby-davidbailey00-updateme)) so I highly doubt anyone is already using it in their own projects. Thoughts on this?

Reply:
Ah yeah I forgot we marked it as private. It doesn't matter then, we don't need to to worry about that.